client/keys/mnemonic.go

```diff
@@ -64,12 +65,16 @@ func MnemonicKeyCommand() *cobra.Command {
 			if err != nil {
 				return err
 			}
-
+			indiscreet, _ := cmd.Flags().GetBool(flagIndiscreet)
+			if !indiscreet {
+				return printDiscreetly(client.GetClientContextFromCmd(cmd), cmd.ErrOrStderr(), "**Important** write this mnemonic phrase in a safe place. Do not share it to anyone.", mnemonic)
+			}
 			cmd.Println(mnemonic)
 			return nil
 		},
 	}
 
 	cmd.Flags().Bool(flagUserEntropy, false, "Prompt the user to supply their own entropy, instead of relying on the system")
+	cmd.Flags().Bool(flagIndiscreet, false, "Print mnemonic directly on current terminal")
```

Comment:
I think the default value and the flag description don't line up. By default, the mnemonic will be displayed.

Reply:
> I think the default value and the flag description don't line up. By default, the mnemonic will be displayed.

Default behaviour shoud be displaying mnemonic on an alternate screen IMO, as this is more safe than displaying directly on current terminal. And if we provide "--indiscreet", this means, "Print mnemonic directly on current terminal"

Reply:
Ahh yes, you're right. I didn't see the `printDiscreetly` call. Thanks!